### PR TITLE
[WIP][Documentation] Task configurator page update - labels shape description added

### PR DESCRIPTION
**Pull request by mdacoca**
Create annotation task page updated for Basic configurator:
- Procedures updated
- Label shape description added
- Screenshots updated
- Typos and proofreading.

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md [hunk 1]
```diff
@@ -1,130 +1,178 @@
...
+   ![Select project](/images/image193.jpg)
+
+   > **Note:** Further steps are valid if the task does not belong to the project. If the task is in
+   > project, the project's labels will be applied to the task.
+
+3. On the **Constructor** tab, click **Add label**, the label constructor menu will open:
+
+   ![Label constructor](/images/image124.jpg)
+
+4. In the **Label name** field, enter the name of the label.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/creating_an_annotation_task.md:39]: Maybe to insert here what label is in our tool. (Labels aka classes object classes to be annotated according to specification).. Or maybe you can suggest better description

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md [hunk 2]
```diff
@@ -1,130 +1,178 @@
...
+5. (Optional) To limit the use of the label to a certain [shape tool](/docs/manual/basics/controls-sidebar/#shapes),
+   from the **Label shape** drop-down select the shape.
+6. (Optional) Select the color for the label.
+
+   ![label shape and color](/images/label_shape.jpg)
 
-The name of the task to be created.
+7. (Optional) Click **Add an attribute** to add [attribute](#add-an-attribute) and its properties.
+8. Click [**Select files**](#select-files).
+9. Click **Continue** to save the label and start adding a new one
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/creating_an_annotation_task.md:48]:
```suggestion
9. Click **Continue** to submit the label and start adding a new one
```

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md [hunk 3]
```diff
@@ -1,130 +1,178 @@
...
+   <br> **Cancel** will reset all added settings.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/creating_an_annotation_task.md:49]:
```suggestion
   <br> **Cancel** will abort current label and return you to labels list.
```

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md [hunk 4]
```diff
@@ -1,130 +1,178 @@
...
+    or **Submit and continue**, to submit the configuration and start a new task.
 
-![](/images/image005.jpg)
+### Label shape
 
-### Projects
+**Label shape** limits the use of the label to certain [shape tool](/docs/manual/basics/controls-sidebar/#shapes).
 
-The project that this task will be related with.
+`Any` - is the default shape, that does not limit labels.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/creating_an_annotation_task.md:57]:
```suggestion
`Any` - is the default shape, that does not limit shape types.
```

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md [hunk 5]
```diff
@@ -1,130 +1,178 @@
...
+  per line). If you upload a video or dataset with images and select the **Use cache** option, you
+  can attach a `manifest.json` file.
+  <br> On how to prepare the `manifest.json` file, see [Dataset manifest.](/docs/manual/advanced/dataset_manifest/)
+- To upload files from cloud storage, go to the **Cloud Storage** tab. Type the cloud storage name,
+  and after that choose the manifest file and select the required files.
+  <br> For more information, see [Attach cloud storage](/docs/manual/basics/attach-cloud-storage/)
+
+### RAW
+
+The **Raw** is a way of working with labels for an advanced user.
```
Comment (bsekachev) [inline site/content/en/docs/manual/basics/creating_an_annotation_task.md:129]: Let's add here: 

**Important note:**

**Be careful** with changing raw specification of an existing task/project. Removing any "id" properties will lead to losing existing annotations. **This property will be removed automatically from any text you insert to this field.**

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md (hunk 1 above)
Comment (mdacoca) [reply site/content/en/docs/manual/basics/creating_an_annotation_task.md:39]: Added it but to the Labels Shape description

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md (hunk 5 above)
Comment (mdacoca) [reply site/content/en/docs/manual/basics/creating_an_annotation_task.md:131]: added

**Comment by mdacoca** [comment]
too messy, closed to redo

**Pull request closed without merging**